### ISSUE TITLE
Expanding Image on Header Grid

### ISSUE THREAD

**Issue by riannazandbelt** [riannazandbelt/components-on-a-grid#1]
Hi @hellachella,

I'm having an issue expanding the .pano image across the grid to cover both columns. Any advice as to which element to target when trying to span the image across columns?

Thanks,
Rianna 

![image](https://user-images.githubusercontent.com/55094049/93628246-3a8d7500-f9b4-11ea-8d25-2d6dbd796a12.png)

**Comment by hellachella** [comment]
Let me look at it.

**Comment by hellachella** [comment]
Paste the following into your editor, I have added code comments to follow along with. 

```
html {
  box-sizing: border-box;
  line-height: 1.5;
  -moz-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -webkit-text-size-adjust: 100%;
  text-size-adjust: 100%;
  font-family: 'Patua One'
}

*,
*::before,
*::after {
  box-sizing: inherit;
}

img {
  width: 100%;
}



header {
  /* Remove the <header> box and allow us to directly apply header’s children to the grid. */
  display: contents;
}

body {
  margin: 1rem;
  /*
  this is where you use display: grid and set up your columns. 
  Use grid-template-area to achieve the layout
  */
}

/* 
.pano {
use this area to place the banner into the grid using grid-template-column 
(the CSS that moves our cells around where we want them) as well as difining 
a grid-area spot (names must match those set up in grid-template-areas). 
stretch it across both top columns using grid-column.
}


.logo {
  use this area to place the logo into the correct column and row and pull it 
  out of its own cell so that it is overlaid on the .pano
} 
*/

ul {
  list-style-type: none;
  margin: 0;
  padding: 0;
  font-size: 2rem;
}

li a {
  color:
}

h2 {
  color: #30658d;
}

p {
  color: #000;
  font-family: serif;
}

header {
  display: grid;
  grid-template-columns: repeat(2, 1fr);
  grid-template-rows: 2fr 4rem;
  padding-bottom: 1rem;
}

h1 {
  grid-row: 2;
  grid-column: 1;
}

.pano {
  grid-column: 1 / -3;
}

main {
  display: grid;
  grid-template-columns: repeat(2, 1fr);
  column-gap: 1rem;
}

figure {
  margin: 0;
}

```

**Comment by riannazandbelt** [comment]
@hellachella 

Great, thank you so much! I have two other quick questions:

1. When I run the assignment through Markbot, I get a message saying that it expected a different value in the font family. I tried playing around with the double vs single quotation marks and removing the sans-serif portion to no avail.

2. Also I tried moving the figures in the MAIN section to each their respective column and rows by targeting each figure through the figure:nth child(1) and figure:nth child(2) (also tried targeting main:nth child (1)), but either both stayed in the same place or both moved to the other column. Is there another way to target just one figure to move it over to the second column?

Thank you again! :) 

![image](https://user-images.githubusercontent.com/55094049/93637192-2ac95d00-f9c3-11ea-842b-405f5204267e.png)
